README.md

```diff
@@ -1,5 +1,14 @@
 # Vue 3 + TypeScript + Vite
 
-This template should help get you started developing with Vue 3 and TypeScript in Vite. The template uses Vue 3 `<script setup>` SFCs, check out the [script setup docs](https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup) to learn more.
+## Usage
 
-Learn more about the recommended Project Setup and IDE Support in the [Vue Docs TypeScript Guide](https://vuejs.org/guide/typescript/overview.html#project-setup).
+### Mock
+
+`npm install`
+`json-server db.json --watch --delay=500`
```

Comment:
I had to switch to different API because the previous one does not provide forecast data for the free tier and the new one has limit of 50 requests/day so I use local db for dev and then I switch to external API only to see if everything works fine